src/app/shared/components/form-required-mark/form-required-mark.component.html

```diff
@@ -0,0 +1 @@
+*
```

Comment:
It can be considered whether we need a file for this or if it can be set in the `@Component` template in the TypeScript file.

Reply:
Removed

Reply:
> It can be considered whether we need a file for this or if it can be set in the @Component template in the TypeScript file.

I would advise against that. We follow a pattern where we have the html in their separate file. You are suggesting to break that pattern.

Reply:
@LindvedKrvang @KvelaGorrrrnio What to do here? With or without template?

Reply:
> I would advise against that. We follow a pattern where we have the html in their separate file. You are suggesting to break that pattern.

There is no HTML – just a single character in a string. But either way is fine for me. I'll let it be up to @LindvedKrvang.

Reply:
Revert to the template